src/Histogram.ts

```diff
@@ -27,6 +27,15 @@ const LUT_ARRAY_LENGTH = LUT_ENTRIES * 4;
  * @property {Array.<number>} lut LUT_ARRAY_LENGTH element lookup table as array (maps scalar intensity to a rgb color plus alpha)
  * @property {Array.<ControlPoint>} controlPoints
  */
+type ControlPoint = {
+  x: number;
+  opacity: number;
+  color: [number, number, number];
```

Comment:
Just noticed that this type for `color` doesn't match what's in the JSDoc block in line 22

Reply:
will fix it.  I wonder if jsdoc could make documentation from types directly...